BeeSwift/TimerViewController.swift

```diff
@@ -90,6 +98,16 @@ class TimerViewController: UIViewController {
         resetButton.setTitle("Reset", for: .normal)
     }
     
+    override func viewWillDisappear(_ animated: Bool) {
```

Comment:
In addition to when leaving the screen, should we disable the brightness override when the app is moved to background?